docs/guides/integrate-with-ory-cloud-through-webhooks.mdx

```diff
@@ -27,6 +27,11 @@ automatically add a new lead to your CRM when a user registers in your applicati
 You can use webhooks to modify user data when certain events occur. For example, you can use webhooks to update user data in your
 application when a user updates their profile information in another system.
 
+**Passing additional data on registration**
+
+When building your own UI you can pass additional data when completing the registration flow that gets forwarded to any configured
```

Comment:
Can this only be used with a custom UI?

Reply:
It can only be used, when you use the SDK to build a signup, but not with our hosted UI.